example/dns.nomad

```diff
@@ -0,0 +1,27 @@
+
+job "dns" {
+  datacenters = ["dc1"]
+
+  group "dns-group" {
+
+    network {
+        dns {
+            servers = ["127.0.0.1", "127.0.0.2"]
```

Comment:
Can we also update the test to have `options` and `searches`?

```
network {
  dns {
    servers = ["127.0.0.1", "127.0.0.2"]
    searches = ["internal.corp"]
    options = ["ndots:2"]
  }
}
```

Reply:
added.